scripts/gen-uuid-reg.py

```diff
@@ -31,6 +31,10 @@ def emit_uuid_rec(uu, sym):
     uuidinit = "{ " + ", ".join(wrecs) + ", { " + ", ".join(byts) + " } }"
     out_recs.append(f"#define _UUIDREG_{sym} {uuidinit}")
 
+    uuidstr = uu[:23] + '-' + uu[23:]
+    out_recs.append(f'#define UUIDREG_STR_{sym.upper()} "{uuidstr}"')
```

Comment:
not that important, but maybe we should make it more like the original version, e.g. `_UUIDREG_STR_DRC` or actually even `_UUIDREG_STR_drc`

Reply:
`_UUIDREG_` values are used only by `SOF_DEFINE_REG_UUID` macro. `UUIDREG_STR_` values ​​are intended for use in toml files. I think they will look better without the `_` prefix, which may suggest that it is a private value.